### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#111249] [Other Feature Feedback, Questions, & Ideas] [answered]
##General

##Account Suspension 

My account was suspended for no valid reason also I didn't received any suspension email for account "0xaira". So I have created a new account to continue my open-source. Now when I am trying to push code to my new account it's showing :
remote: Your account is suspended. Please visit https://support.github.com for more information.
unable to access 'https://github.com/airajena/portfolio.git/': The requested URL returned error: 403

This is really pathetic experience by GitHub support. They are not fixing my account suspension issues . I have raised ticket. Also you are not allowing to create any account from my laptop. Maybe you have blacklisted my IP address. I request you to please fix this. I am already depressed seeing my previous account ban. It's really hurting.  I can't take this anymore 

Suspended Account: https://github.com/0xaira 
![Screenshot (40)](https://github.com/community/community/assets/162343614/5e73e9d3-c0c1-4940-b09f-800dccff9aca)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @airajena, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
